modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/viewmodel/PodcastListViewModel.kt

```diff
@@ -134,7 +134,10 @@ class PodcastListViewModel @Inject constructor(
 
     fun findOrDownloadEpisode(discoverEpisode: DiscoverEpisode, success: (episode: PodcastEpisode) -> Unit) {
         podcastManager.findOrDownloadPodcastRx(discoverEpisode.podcast_uuid)
-            .flatMapMaybe { episodeManager.findByUuidRx(discoverEpisode.uuid) }
+            .flatMapMaybe {
+                @Suppress("DEPRECATION")
+                episodeManager.findByUuidRx(discoverEpisode.uuid)
+            }
```

Comment:
I like the way you have added `@Deprecated` to the method. 
Is it a step too far to change the code to something like the following:
```suggestion
            .flatMapMaybe {
                rxMaybe {
                    episodeManager.findByUuid(discoverEpisode.uuid)
                }
            }
```

Reply:
`rxMaybe` works great there. Thanks!